build/localnet/README.md

```diff
@@ -0,0 +1,165 @@
+# Kubernetes LocalNet <!-- omit in toc -->
+
+This guide shows how to deploy a LocalNet using [pocket-operator](https://github.com/pokt-network/pocket-operator).
+
+- [Dependencies](#dependencies)
+  - [Enabling Kubernetes](#enabling-kubernetes)
+- [LocalNet](#localnet)
+  - [Starting LocalNet](#starting-localnet)
+  - [Viewing Logs](#viewing-logs)
+  - [Terminal Logs](#terminal-logs)
+  - [Tilt Web UI Logs](#tilt-web-ui-logs)
+  - [Scaling Actors](#scaling-actors)
+  - [Stopping \& Clean Resources](#stopping--clean-resources)
+  - [Interacting w/ LocalNet](#interacting-w-localnet)
+    - [Make Targets](#make-targets)
+- [How does it work?](#how-does-it-work)
+- [Troubleshooting](#troubleshooting)
+  - [Why?](#why)
+  - [Force Trigger an Update](#force-trigger-an-update)
+  - [Force Restart](#force-restart)
+  - [Full Cleanup](#full-cleanup)
+  - [Docker Desktop](#docker-desktop)
+- [How to change configuration files](#how-to-change-configuration-files)
+- [Code Structure](#code-structure)
+
+## Dependencies
+
+1. [tilt](https://docs.tilt.dev/install.html)
+   - Note: automatically installed when running `make install_cli_deps`
+2. `Kubernetes cluster`: [installation options](https://docs.tilt.dev/choosing_clusters.html)
+3. `kubectl`: CLI is required and should be configured to access the cluster. This should happen automatically if using Docker Desktop, Rancher Desktop, k3s, k3d, minikube, etc.
+4. `helm`: required to template the yaml manifests for the dependencies (e.g. postgres, grafana). Installation instructions available [here](https://helm.sh/docs/intro/install).
+
+### Enabling Kubernetes
+
+You may need to manually enable Kubernetes if using Docker desktop:
+
+![Docker desktop kubernetes](https://user-images.githubusercontent.com/1892194/216165581-1372e2b8-c630-4211-8ced-5ec59b129330.png)
+
+## LocalNet
+
+### Starting LocalNet
+
+```bash
+make localnet_up
+```
+
+### Viewing Logs
+
+The developer can then view logs either from a browser or terminal.
+
+### Terminal Logs
+
+- `make localnet_logs_validators` - shows prior logs
+- `make localnet_logs_validators_follow` - shows prior logs and follows the new log lines as validators do their work
+
+### Tilt Web UI Logs
+
+- Pressing `space` in terminal where you started `tilt`
+- Go to [localhost:10350](http://localhost:10350/)
+
+![tilt UI](https://user-images.githubusercontent.com/1892194/216165833-b9e5a98c-87a8-4355-87c9-0420a8a598bf.png)
+
+### Scaling Actors
+
+When starting a k8s LocalNet, `localnet_config.yaml` is generated (with default configs) in the root of the repo if doesn't already exist.
+
+The config file can be modified to scale the number of actors up/down. As long as `localnet_up` is running, the changes should be automatically applied within seconds.
+
```

Comment:
Could you add a GIF or video here?

Reply:
uploading it here, will include in readme file.

https://user-images.githubusercontent.com/4950477/216490690-8ac4c16a-25e1-4202-b2e5-03215030c82c.mp4

Reply:
I don't think there's a way to embed videos in readme, huh..